_config.yml

```diff
@@ -15,7 +15,7 @@ email:
     contact_formspree:  xgkblajx
     contact_cc:         "contact@bigchaindb.com,kamal@bigchaindb.com,ricardo@bigchaindb.com,gautam@bigchaindb.com,bilel@bigchaindb.com"
     cla:                xwojnopx
-    cla_cc:             troy@ascribe.io
+    cla_cc:             troy@ascribe.io,upod-team@bigchaindb.com
```

Comment:
Should the list of email addresses be inside double quotes? It is for `contact_cc` and `services_cc`.

Reply:
yes, actually makes the string conversion more safe